scripts/shared-vars-public.sh

```diff
@@ -108,4 +108,4 @@ SUPPORTED_SERVICES=(
 
 # Regex used to match long running tests cases (10m+ runtime). Any new
 # long-running tests should be added to this regex in alphabetical order.
-LONG_RUNNING_CRUD_TESTS_REGEX="basicalloydbbackup|basicalloydbinstance|basicalloydbsecondarycluster|basicalloydbsecondaryinstance|cidrconnector|cloudidsendpoint|configcontrollerinstance|containercluster|containernodepool|databasealloydbuser|datafusioninstance|filestorebackup|filestoreinstance|fullalloydbbackup|fullalloydbcluster|gkehubfeaturemembership|gkehubmembership|iamalloydbuser|postgresinstance|readalloydbinstance|restorebackupalloydbcluster|zonalalloydbinstance|memcacheinstance|redisinstance|removedefaultnodepool|sqlinstanceencryptionkey|subnetconnector|vertexaidatasetencryptionkey|vertexaiendpointencryptionkey"
+LONG_RUNNING_CRUD_TESTS_REGEX="basicalloydbbackup|basicalloydbinstance|basicalloydbsecondarycluster|basicalloydbsecondaryinstance|cidrconnector|cloudidsendpoint|configcontrollerinstance|containercluster|containernodepool|databasealloydbuser|datafusioninstance|filestorebackup|filestoreinstance|fullalloydbbackup|fullalloydbcluster|gkehubfeaturemembership|gkehubmembership|iamalloydbuser|postgresinstance|pscalloydbinstance|readalloydbinstance|restorebackupalloydbcluster|zonalalloydbinstance|memcacheinstance|redisinstance|removedefaultnodepool|sqlinstanceencryptionkey|subnetconnector|vertexaidatasetencryptionkey|vertexaiendpointencryptionkey"
```

Comment:
What about pscalloydbcluster test? I assume it also takes more than 10 minutes to finish?

Reply:
It doesn't, takes less than 4 minutes